packages/kbn-unified-data-table/README.md

```diff
@@ -0,0 +1,3 @@
+# @kbn/unified-data-table
+
+Empty package generated by @kbn/generate
```

Comment:
Sorry, it took me while to make it nice. But finally I added [this commit](https://github.com/elastic/kibana/pull/163211/commits/5be3eaf159f6468466a6dc0c4744316a7801c222) with the missing content. I hope this will be helpful for functionality understanding for the future package consumers and maybe easier maintenance.

Reply:
Wow! this is not nice! this is very nice! Great content 👍